Lesson_14/logger.py

```diff
@@ -0,0 +1,27 @@
+"""Create logger."""
```

Comment:
For short tasks please try to organize logger and logging within one file. It will save my time for hometask check )
Thx in advance

Reply:
Update files according to suggestions in the commit 62ad0d2a3577106552bdf0d35792148b01cf512c